### ISSUE TITLE
Command vowel dummy replacements 

### ISSUE THREAD

**Issue by Dekker1** [valentjn/vscode-ltex#366]
Note: Per the contribution guidelines, deleting parts of the template or not filling in vital information may result in the issue to be immediately closed as invalid.

**Is your feature request related to a problem? Please describe.**

Currently, commands can be replaced by dummy values, as described in the documentation https://valentjn.github.io/vscode-ltex/docs/settings.html#ltexlatexcommands. However, I have some commands that are replaced by words starting with a vowel:
```
This sentence shows an \instance{} of my problem.
```
The problem, however, is that when using dummy replacement the LanguageTool backend now warns that "an" should be replaced by "a".
Although this by itself is not ideal, it can be easily ignored. I'm, however, afraid that this might make me miss certain cases cases where I've used "a" instead of "an", which seems to happen to me a lot.

**Describe the solution you'd like**
To me the simplest solution would be to have two additional options for command replacement that replace the command using a dummy value starting with a vowel, and a plural version that does the same.

**Describe alternatives you've considered**
This issue is closely related to: https://github.com/valentjn/vscode-ltex/issues/43. There another consideration was allowing the user to decide on the dummy text to be used.

**Additional context**
N/A

**Comment by Dekker1** [comment]
In further use, I also found that I have some macros that might result in a vowel depending on their arguments.
An example is the `\gls{}` command from the glossaries package, whose command generally get replaced by their argument.

```
This sentence is an \gls{instance} of a \gls{problem}
```

For these commands just adding `VowelDummy` wouldn't be enough, since it depends on the argument of the command.
I'm wondering if it would be possible to consider a different matching method where you could match arguments depending on a certain regular expression.

**Comment by valentjn** [comment]
Thanks for the suggestion. There is now the possibility to use `"vowelDummy"` for [`ltex.latex.commands`](https://valentjn.github.io/vscode-ltex/docs/settings.html#ltexlatexcommands) and [`ltex.markdown.nodes`](https://valentjn.github.io/vscode-ltex/docs/settings.html#ltexmarkdownnodes).

As for the second comment, you can just use `{"ltex.latex.commands": {"\\gls{}": "default"}}` for this.

**Comment by valentjn** [comment]
Feature released in 11.0.0.
